docs/quest_template.md

```diff
@@ -251,347 +377,253 @@ Like all flag based fields, **QuestFlags** can be added for the different types
+
+### RewardItem2
+
+[item Id 2](item_template#entry) given for reward (no choice).
+
+### RewardAmount2
+
+Amount to be obtained from the above-mentioned item
+
+### RewardItem3
+
+[item Id 3](item_template#entry) given for reward (no choice).
+
+### RewardAmount3
+
+Amount to be obtained from the above-mentioned item
+
+### RewardItem4
+
+[item Id 4](item_template#entry) given for reward (no choice).
+
+### RewardAmount4
+
+Amount to be obtained from the above-mentioned item
+
+### ItemDrop1
+
+
+
+### ItemDropQuantity1
+
+
+
+### ItemDrop2
+
+
+
+### ItemDropQuantity2
+
+
+
+### ItemDrop3
+
+
+
+### ItemDropQuantity3
 
-Will Give X Arena Points to the player that completes the quest. Leave"0"For no Arena Points Reward.
 
-### RewardChoiceItemId
 
-Id of item available for reward choice.
-Number of Charges in rewarded item available.
+### ItemDrop4
 
-### RewardChoiceItemCount
 
-Will choose how many items will be added for reward. E.g "RewardChoiceItemId" is the reward item and "RewardChoiceItemCount" is the count of how many items will be added as a reward.
 
-### RewardItemId
+### ItemDropQuantity4
 
-[item Id](item_template#entry) given for reward (no choice).
 
-### RewardItemCount
 
-`field-no-description|46`
+### RewardChoiceItemID1
 
-### RewardFactionId
 
-Faction Id (from [Faction.dbc](Faction)) for which the quest give reputation points.
-Number of gain or lost reputation points for Faction at quest completion. This is special reputation rewarding. Normal reputation reward to quest rewarding creature faction calculated and added automatically.
 
-### RewardFactionValueId
+### RewardChoiceItemQuantity1
 
-This field is used for reputation lookup in [QuestFactionReward.dbc](QuestFactionReward) if [RewardFactionValueId](#quest_template-RewardFactionValueId) is 0. Value X in this field indicates RepX column of [QuestFactionReward.dbc](QuestFactionReward). If RewardRepValueId is positive, reputation from the first row of QuestFactionReward.dbc will be used, for negative values the second row is used.
 
-### RewardFactionValueIdOverride
+### RewardChoiceItemID2
+
+
+### RewardChoiceItemQuantity2
+
+
+### RewardChoiceItemID3
+
+
+### RewardChoiceItemQuantity3
+
+
+### RewardChoiceItemID4
+
+
+### RewardChoiceItemQuantity4
+
+
+### RewardChoiceItemID5
+
+
+### RewardChoiceItemQuantity5
+
+
+### RewardChoiceItemID6
+
+
+### RewardChoiceItemQuantity6
 
-This field is used to give reputation values not present in QuestFactionReward.dbc or to override them if [RewardRepValueId](#quest_template-RewardRepValueId) is wrong for some reason. The value in this field is 100× the intended reputation reward (if you want to give 400 rep, put 40000 in RewardFactionValueIdOverride).
 
 ### POIContinent
 
-MapId of a quest point of interest (POI - Point Of Interest). POI will be shown on the map when quest is active.
 
 ### POIx
 
-X coordinate of quest POI.
 
 ### POIy
 
-Y coordinate of quest POI.
 
 ### POIPriority
 
-TODO
```

Comment:
Why are we just removing this information?

Reply:
> Why are we just removing this information?

I really don't know, but now I'm going to check it and restore it. Maybe it was a typo.